python/nav/web/api/v1/views.py

```diff
@@ -206,7 +206,37 @@ def _build_object(checker):
         }
 
 
-class RoomViewSet(NAVAPIMixin, viewsets.ReadOnlyModelViewSet):
+class LoggerMixin(object):
```

Comment:
How about audit logging? If we're serious about the auditlogging system, we should use it more...

Reply:
Good call, on list for next week if time